drivers/gpio/gpio_tca9538.c

```diff
@@ -0,0 +1,536 @@
+/*
+ * Copyright (c) 2018 Peter Bigot Consulting, LLC
+ * Copyright (c) 2018 Aapo Vienamo
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ * Copyright (c) 2019 Vestas Wind Systems A/S
+ * Copyright (c) 2020 ZedBlox Ltd.
+ * Copyright (c) 2021 Laird Connectivity
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT ti_tca9538
```

Comment:
Let's move the define just above 
```
DT_INST_FOREACH_STATUS_OKAY(GPIO_TCA9538_INIT)
```
line. It will make it easier to support other chips with different 'compatible' properties in the future.

Reply:
Define moved.